docs/shared/register-federated-cli.mdx

```diff
@@ -1,53 +1,42 @@
+graph LR;
   subgraph " "
   gateway([Gateway]);
-  serviceA[Products service];
-  serviceB[Reviews service];
-  serviceC[Inventory service];
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  serviceC[Inventory subgraph];
   end
   registry{{Apollo Schema Registry}};
   gateway --- serviceA & serviceB & serviceC;
-  serviceA -. --serviceName=Products  .-> registry;
-  serviceB -. --serviceName=Reviews  .-> registry;
-  serviceC -. --serviceName=Inventory  .-> registry;
+  serviceA & serviceB & serviceC -. rover subgraph publish  .-> registry;
   class registry secondary;
 ```
 
-First, <a href="https://www.apollographql.com/docs/devtools/cli/" target="_blank">install the Apollo CLI</a> if you haven't yet.
+If you haven't yet:
 
-Then, **do the following for each of your implementing services**:
+*  <a href="https://www.apollographql.com/docs/rover/getting-started/" target="_blank">Install the Rover CLI.</a>
+* <a href="https://www.apollographql.com/docs/rover/configuring/" target="_blank">Authenticate Rover</a> with Apollo Studio.
 
-1. Obtain the following values, which are required for the `apollo service:push` command:
+Then, **do the following for each of your subgraphs**:
 
-    * The URL that your federated gateway will use to communicate with the service (e.g., `http://products-graphql.svc.cluster.local:4001/`).
-    * The name that uniquely identifies the service within your graph (e.g., `products`).
-    * The service's schema. The Apollo CLI can either:
+1. Obtain the following values, which are required for the `rover subgraph publish` command:
+
+    * The name that uniquely identifies the subgraph within your graph (e.g., `products`).
+    * The URL that your gateway will use to communicate with the subgraph (e.g., `http://products-graphql.svc.cluster.local:4001/`).
+    * The service's schema. The Rover CLI can either:
         * Use a `.gql` or `.graphql` file saved on your local machine, or
         * Perform an introspection query on the running service to fetch the schema
 
-2. Run the `apollo service:push` command, providing the following options (documented below):
+2. Run the `rover subgraph publish` command, providing your schema in one of the ways shown:
 
     ```bash
-    apollo service:push \
-      --localSchemaFile=./schema.gql \ # (or --endpoint)
-      --key=service:docs-example-graph:NYKgCqwfCyYPIm84WVXCdw
-      --graph=docs-example-graph \
-      --variant=local-development \
-      --serviceName=products \
-      --serviceURL=http://products-graphql.svc.cluster.local:4001/
+    # Provide a local .graphql file path
+    rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema ./schema.graphql 
+
+    # Provide an introspection result via stdin
+    rover graph introspect http://localhost:4000 | rover subgraph publish my-graph@my-variant --name products --routing-url http://products-graphql.svc.cluster.local:4001/ --schema -
```

Comment:
We should consider using `subgraph introspect` here, which is a more common case and yields the "service SDL" with federation directives instead of a plain introspection result.